spec/models/tag_spec.rb

```diff
@@ -13,4 +13,9 @@
 RSpec.describe Tag, type: :model do
   it { expect(subject).to respond_to(:id) }
   it { expect(subject).to respond_to(:name) }
+
+  it "is valid with name" do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.